.changeset/loud-deers-run.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-qwik': patch
+---
+
+migration from tsm to ts node
```

Comment:
```suggestion
migration from tsm to tsx
```

Reply:
Also the PR title needs to be adjusted: `build: switching tsm to tsx`. Or something along those lines.

Reply:
well currently we support `FEAT:` for features and `FIX:` for bugs fixes and `DOCS:` for documentation changes. 

Maybe we can add `INFRA:` if you think it's important

but for this PR I think we can merge as it is... WDYT?

Reply:
yes no problem for me :)